package.json

```diff
@@ -56,8 +56,8 @@
     ]
   },
   "devDependencies": {
-    "@aws-amplify/core": "^4.7.8",
-    "@aws-amplify/geo": "^1.3.20",
+    "@aws-amplify/core": "5.0.0",
```

Comment:
We might want this to be more permissive to include updates, not sure if matters immediately since it is just a devDep though

Reply:
Yeah for now just wanted to lock it down but since it's devDep we can easily change it anyways